src/pages/en/reference/api-reference.md

```diff
@@ -379,6 +548,8 @@ export async function getStaticPaths() {
 
 The `getStaticPaths()` function should return an array of objects to determine which paths will be pre-rendered by Astro.
 
+It can also be used in static file endpoints for [dynamic routing](d/en/core-concepts/endpoints/#params-and-dynamic-routing).
```

Comment:
Oops, not sure what happened here, but this should fix it:

```suggestion
It can also be used in static file endpoints for [dynamic routing](/en/core-concepts/endpoints/#params-and-dynamic-routing).
```